presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -471,14 +471,41 @@ public void testSchemaAuthorizationForUser()
                 .setIdentity(Identity.forUser("user").withPrincipal(getSession().getIdentity().getPrincipal()).build())
                 .build();
 
+        Session anotherUser = testSessionBuilder()
```

Comment:
Please have a dedicated test for CREATE SCHEMA WITH AUTHORIZATION. If you mix this with `ALTER SCHEMA` it is getting very complicated.